python-string-formatting/format-method.py

```diff
@@ -0,0 +1,12 @@
+number_template = "The number is {}"
+print(number_template.format(42))
+
+a = 5
+b = 10
+sum_template = "{0} plus {1} is {2}"
```

Comment:
Do we want to move the templates to the top here (and in line 11), similarly to how we did in the tutorial?